client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -68,7 +68,9 @@ const MediaModalImageEditorCanvas = React.createClass( {
 	},
 
 	componentWillReceiveProps( newProps ) {
-		this.getImage( newProps.src );
```

Comment:
I must have missed this in the PR too since the placeholders removed a bunch of src checks. 😄